### PR TITLE
Fix empty lines for implicit concatination

### PR DESCRIPTION
**Pull request by blablatdinov**
# I have made things!

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [x] I have added my changes to the `CHANGELOG.md`

## Related issues

#2787 

🙏 Please, if you or your company is finding wemake-python-styleguide valuable, help us sustain the project by sponsoring it transparently on https://opencollective.com/wemake-python-styleguide. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

**Comment by blablatdinov** [comment]
@sobolevn take a look, please

File: wemake_python_styleguide/visitors/ast/function_empty_lines.py [hunk 1]
```diff
@@ -30,9 +30,8 @@ def _is_target_line(self, token: tokenize.TokenInfo) -> bool:
         is_comment = False
         if stripped_token_line:
             is_comment = '#' in stripped_token_line[0]
```
Comment (sobolevn) [inline wemake_python_styleguide/visitors/ast/function_empty_lines.py:32]: Shouldn't this be `is_comment = '#' == stripped_token_line[0]`?

**Review by sobolevn — COMMENTED** [review]
Thank you! Looks mostly correct

File: wemake_python_styleguide/visitors/ast/function_empty_lines.py (hunk 1 above)
Comment (blablatdinov) [reply wemake_python_styleguide/visitors/ast/function_empty_lines.py]: Yes, you're right. Thanks!

File: wemake_python_styleguide/visitors/ast/function_empty_lines.py [hunk 2]
```diff
@@ -29,10 +29,9 @@ def _is_target_line(self, token: tokenize.TokenInfo) -> bool:
         stripped_token_line = token.line.strip()
         is_comment = False
         if stripped_token_line:
-            is_comment = '#' in stripped_token_line[0]
-        is_string = token.type == tokenize.STRING
+            is_comment = stripped_token_line[0] == '#'
```
Comment (sobolevn) [inline wemake_python_styleguide/visitors/ast/function_empty_lines.py:32]: or even better: 
```suggestion
            is_comment = stripped_token_line.startswith('#')
```

**Review by sobolevn — APPROVED** [review]

**Pull request merged**